<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptBenchmarkDialog</class>
 <widget class="QDialog" name="ScriptBenchmarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script benchmark</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="VSEditorLog" name="feedbackTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="metricsEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="processingProgressBar">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>%v / %m</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="framesLabel">
       <property name="text">
        <string>Frames:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fromFrameSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="toLabel">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="toFrameSpinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="wholeVideoButton">
       <property name="text">
        <string>Full length</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startStopBenchmarkButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSEditorLog</class>
   <extends>QTextEdit</extends>
   <header>../../common-src/log/vs_editor_log.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
